stories/guides/component-dependencies.stories.mdx

```diff
@@ -12,126 +12,64 @@ JET websites make use of `JetSansDigital` as their main typeface (`Regular`, `Bo
+We expect consuming applications to include the optional mixins for global typography and link styles – via `@include typography();` and `@include links();`. Similar to the reset styles, these mixins can only be included once the Fozzie SCSS library has been imported, due to the mixins being reliant on SCSS variables, mixins and functions that are defined there.
 
 ### Utility classes
 
 In specific styling situations, it's useful to have a set of "trump" utility classes; such as to hide or show information for accessibility purposes (for example `.is-visuallyHidden`).
 
-We are working on bringing these utility classes to fozzie v5 in a way that is shareable between components, but this is currently a work in progress – we're currently working on making a set of utility classes available across components, for common situations like needing to hide or show information for accessibility purposes (for example `.is-visuallyHidden`).
+We are working on adjusting these utility classes in a way that is shareable between components without needing definition by the consuming application, but this is currently a work in progress.
+
+Once these have been imported by calling the optional mixin `@include f.trumps-utilities();`, these utility classes will be available for use in components and to consuming applications.
 
-Once these have been defined, these utility classes will be available for use in components and to consuming applications.
 
-## Importing the Fozzie SCSS framework
+## Importing the Fozzie SCSS Helper Library
 
-Including the Fozzie SCSS framework in any application is straightforward, but will likely be subtly different depending on the CLI or bundling tool you are using.
+Including Fozzie SCSS in any application is straightforward, but will likely be subtly different depending on the CLI or bundling tool you are using.
 
 If you are creating a new UI component, the configuration needed to include the base utility variables, mixins and functions will be automatically setup for you when using our Yeoman component generator tool.
 
 ### VueCLI (via webpack-chain)
 
 *_These setup instructions are correct with respect to `sass-loader` v10. Future releases may change the configuration slightly._*
 
-The Fozzie SCSS utility library has a couple of dependencies that need to be included by any consuming application (via NPM or Yarn). These are:
+Fozzie SCSS is dependent only on Sass (and specifically `dart-sass`). To use Fozzie SCSS you will need at a minimum:
 
-- `fozzie` (>= v5.0.0-beta.2)
-- `node-sass-magic-importer` (>= v5.3.2)
-- `eyeglass` (>= v3.0.2)
+- `@justeat/fozzie` (>= v9.0.0)
+- `sass` (>= v1.53.0)
```

Comment:
Is it worth saying to remove change from node-sass > sass here, or do you think the migration guide covers this instead?

Reply:
As this part assumes that someone is new to using the library, then as long as they know to use `v1.53.0`, that should be ok? I think as you mention, the migration guide is targetted at those using `node-sass`, so that should cover off anyone coming from that side too.